src/ralph/security/views.py

```diff
@@ -1,26 +1,43 @@
 # -*- coding: utf-8 -*-
+import logging
+
 from django.core.exceptions import ObjectDoesNotExist
-from django.db.models import Count
+from django.core.urlresolvers import NoReverseMatch
 from django.utils.safestring import mark_safe
 from django.utils.translation import ugettext_lazy as _
 
 from ralph.admin import RalphAdmin, register
+from ralph.admin.helpers import get_admin_url
+from ralph.admin.sites import ralph_site
 from ralph.admin.views.extra import RalphDetailView
 from ralph.security.models import Vulnerability
 
 
+logger = logging.getLogger(__name__)
+
+
+def _url_name_for_change_view(obj, view_name):
```

Comment:
could we cache it (based on `type(obj)` and `view_name`)? use for example `django.utils.lru_cache`